.github/workflows/reusable.yml

```diff
@@ -80,12 +80,12 @@ jobs:
               run: |
                 source $(poetry env info --path)/bin/activate
                 poetry check
-
             - name: Freeze flask
               run: |
                source $(poetry env info --path)/bin/activate
                echo "Installing the ONS design system"
                 ./get_design_system.sh
+                ./generate_mk_docs.sh
```

Comment:
I think it could be either, everything is generated via mkdocs, even external markdown and that shell script only handles running mkdocs and moving the built files into the relevant directories for the ons design system